source/documentation/deploying_apps/deploying_django.md

```diff
@@ -27,37 +27,20 @@ Before deploying a Django app, you must:
 
 4. Make sure you have all the required modules for your project installed into your virtual environment (including Django).
 
-1. Generate a ``requirements.txt`` file if your project doesn't already have one by running ``pip freeze > requirements.txt`` in the root of the local folder.
-    Add the following lines to the ``requirements.txt`` file.
+1. Generate a `requirements.txt` file if your project doesn't already have one by running `pip freeze > requirements.txt` in the root of the local folder.
 
-        whitenoise==1.0.6  #manages static assets
-        waitress==0.8.9 #a pure python WSGI server that is a replacement for gunicorn
+    Add the following lines to the `requirements.txt` file.
 
-4. Edit your `wsgi.py` file.
+        whitenoise==5.2.0  # manages static assets
+        waitress==2.0.0    # a pure python WSGI server that is a replacement for gunicorn
 
-    When you create a Django project, a default `wsgi.py` file should be created for you in the project folder. Excluding the opening comments, the default `wsgi.py` looks like this:
+4. Tell Django to use `whitenoise` to serve static files. In `settings.py` within the project folder, add this line to `MIDDLWARE`, immediately after `middleware.security.SecurityMiddleware`.
```

Comment:
```suggestion
4. Tell Django to use `whitenoise` to serve static files. In `settings.py` within the project folder, add this line to `MIDDLWARE`, immediately after `middleware.security.SecurityMiddleware`:
```

Reply:
`s/MIDDLWARE/MIDDLEWARE/`

Reply:
Also, should we link to the whitenoise tutorial page for this? http://whitenoise.evans.io/en/stable/django.html

Reply:
@0atman  Good idea. c65a864cfcc52d0321aec76c9fd0fa09120d3fb4